gobblin-modules/gobblin-kafka-09/src/main/java/org/apache/gobblin/kafka/writer/Kafka09DataWriter.java

```diff
@@ -49,6 +58,8 @@
 @Slf4j
 public class Kafka09DataWriter<D> implements AsyncDataWriter<D> {
 
+  private static final Logger LOG = LoggerFactory.getLogger(Kafka09DataWriter.class);
```

Comment:
@Slf4j already provides a logger (refer lombok), so you need not create it again. Please reuse the same